examples/runExample.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+print_error() {
+    >&2 echo "$@"
+}
+
+CAMERA=$1
+MAPPER=$2
+VALIDATION_DATA=$3
+CONFIG_FILE=$4
+
+if [[ $# -gt 4 ]]
+then
+    INGEST=$5
+    PROCESSCCD=$6
+else
+    INGEST=ingestImages.py
+    PROCESSCCD=processCcd.py
+fi
+
+
+PRODUCT_DIR=${VALIDATE_DRP_DIR}
+# OS X El Capitan SIP swallows DYLD_LIBRARY_PATH so export the duplicate in LSST_LIBRARY_PATH
+if [[ -z "$DYLD_LIBRARY_PATH" ]]; then
+    export DYLD_LIBRARY_PATH=$LSST_LIBRARY_PATH
+fi
+
+WORKSPACE=${CAMERA}
+if [ -d "${WORKSPACE}" ]; then
+   rm -rf "${WORKSPACE}"
+fi
+
+# cleanup and create directories
+echo "Ingesting Raw data"
+INPUT=${WORKSPACE}/input
+mkdir -p "$INPUT"
+OUTPUT=${WORKSPACE}/output
+mkdir -p "$OUTPUT"
+
+echo "$MAPPER" > "${INPUT}"/_mapper
+
+# ingest raw data
+RAWDATA=${VALIDATION_DATA}
+${INGEST} "${INPUT}" "${RAWDATA}"/*.fz --mode link
+
+# Set up astrometry 
+export ASTROMETRY_NET_DATA_DIR="${VALIDATION_DATA}"/../astrometry_net_data
+
+# Create calexps and src
+echo "running processCcd"
+MACH=$(uname -s)
+if [ "$MACH" == Darwin ]; then
+    NUMPROC=$(sysctl -a | grep machdep.cpu | grep core_count | cut -d ' ' -f 2)
+else
+    NUMPROC=$(grep -c processor /proc/cpuinfo)
+fi
```

Comment:
As you've defined it above, I think `NUMPROC` will be the number of physical cores on a Mac and the number of virtual (hyperthreaded) cores on Linux. I suspect the difference is accidental. You could make them consistent (and simplify the above) by using `NUMPROC=$(sysctl -n hw.npcu)` on Darwin. (You could also use `hw.physicalcpu` and `hw.logicalcpu` to be really clear about which you want).